front/conditionvalues.php

```diff
@@ -0,0 +1,23 @@
+<?php
+	include ('../../../inc/includes.php');
+	
+	if(isset($_GET['id'])) {
+		$conditionValuesResult = $DB->request([
+			'SELECT' => ['default_values', 'glpi_plugin_formcreator_questions.values'],
+			'FROM'   => 'glpi_plugin_formcreator_questions',
+			'WHERE'  => [
+				'id' => $_GET['id']
+			]
+		])->next();
+		$conditionValues = $conditionValuesResult['values'];
```

Comment:
starting from this line, I think this code should go in a class. Please have a look at the structure of front/form.fform.php as an example.